src/Websites/Websites/help/Update-AzWebAppTrafficRouting.md

```diff
@@ -151,4 +151,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Get-AzWebAppTrafficRouting](./Get-AzWebAppTrafficRouting.md)
 
-[Remove-AzWebAppTrafficRouting](./Remove-AzWebAppTrafficRouting.md)
```

Comment:
Could you revert unnecessary change

Reply:
Yep! I didn't realize that change was in there!